Makefile

```diff
@@ -119,13 +119,10 @@ docker-push: ## Push docker image with the manager.
 PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
 .PHONY: docker-buildx
 docker-buildx: ## Build and push docker image for the manager for cross-platform support
-	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
```

Comment:
I don't understand the purpose of those deleted lines but I see that such logic is still present in KIM. Could you confirm that this deletion is intentional?